src/components/LoggedIn/LoggedIn.jsx

```diff
@@ -2,7 +2,9 @@ import "./LoggedIn.scss";
 import dropDown from "../../assets/images/dropDown.svg";
 import profilePic from "../../assets/images/profilePic.svg";
 
-const LoggedIn = ({firstName, lastName}) => {
+const LoggedIn = ({firstName="Samantha", lastName="Brooks"}) => {
```

Comment:
do we want this hardcoded?

Reply:
This is just a placeholder. We dont current have the name on the firebase

Reply:
cool, maybe add a comment above stating this

Reply:
all done :)